src/azure-cli/azure/cli/command_modules/acs/custom.py

```diff
@@ -2966,6 +2974,16 @@ def aks_update(cmd, client, resource_group_name, name,
             tier="Free"
         )
 
+    if disable_local_accounts and enable_local_accounts:
+        raise CLIError('Cannot specify --disable-local-accounts and --enable-local-accounts '
+                       'at the same time.')
```

Comment:
Could you please use the specific error type `MutuallyExclusiveArgumentError` instead of `CLIError`?

Reply:
Thanks for reviewing, updated.